plugins/action/action.h

```diff
@@ -246,7 +246,28 @@ class Action : public PluginBase {
      */
     float get_max_speed_m_s() const;
 
-    // Non-copyable
+    /**
+     * @brief Set the return to launch minimum return altitude.
+     *
+     * When return to launch is initiated, the vehicle climbs to the return altitude if it is lower
+     * and stays at the current altitude if higher than the return altitude. Then it returns to the
+     * home location and lands there.
+     *
+     * @param relative_altitude_m Return altitude relative to takeoff location, in meters.
+     */
+    void set_return_to_launch_return_altitude(float relative_altitude_m);
+
+    /**
+     * @brief Get the return to launch minimum return altitude.
+     *
+     * When return to launch is initiated, the vehicle climbs to the return altitude if it is lower
```

Comment:
I might just remove this and do a see also to the `get_max_speed_m_s` (or visa versa)

Reply:
Right.